src/Okta.Idx.Sdk.UnitTests/Okta.Idx.Sdk.UnitTests.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net7.0</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
Any reason we're rolling back to .NET 6?

Reply:
Long term support vs. short; also, build environment support.